xyz-hub-service/src/main/java/com/here/xyz/hub/task/FeatureTaskHandler.java

```diff
@@ -1543,11 +1545,41 @@ private static void defineGlobalSearchableField(StatisticsResponse response, Fea
     }
   }
 
-  private static void defineContentUpdatedAtField(StatisticsResponse response, FeatureTask task) {
-    StatisticsResponse.Value<Long> contentUpdatedAtVal = new StatisticsResponse.Value(task.space.contentUpdatedAt);
-    // Due to caching the value of contentUpdatedAt field could be obsolete in some edge cases
-    contentUpdatedAtVal.setEstimated(true);
-    response.setContentUpdatedAt(contentUpdatedAtVal);
+  private static Future<Void> defineContentUpdatedAtField(StatisticsResponse response, FeatureTask.GetStatistics task) {
+    Promise<Void> p = Promise.promise();
+
+    if (task.space.getExtension() != null) {
+      MultiMap queryParams = task.context.get("queryParams");
```

Comment:
Reading / parsing of query params happened in REST tier already. No need to parse the query params on task level again. Instead we could read the context like: `task.getEvent().getContext()`

Reply:
Fixed, thanks.